pyrightconfig.stricter.json

```
@@ -81,7 +87,7 @@
         "stubs/ttkthemes",
         "stubs/vobject",
         "stubs/workalendar",
-        "stubs/wurlitzer",
+        "stubs/wurlitzer"
```

nit: I'd prefer it if we kept the trailing comma ;)

```suggestion
        "stubs/wurlitzer",
```